metaflow/plugins/secrets/__init__.py

```diff
@@ -0,0 +1,14 @@
+import abc
+
+
+class SecretsProvider(abc.ABC):
+    @abc.abstractmethod
+    def get_secret_as_env_vars(self, secret_id, options={}):
```

Comment:
nit: document the return type (Dict[str, str]). Consider renaming the method: `get_secret_as_dict` which is a bit more accurate.